services/web/server/src/simcore_service_webserver/licenses/_licensed_checkouts_api.py

```diff
@@ -0,0 +1,109 @@
+from aiohttp import web
+from models_library.api_schemas_resource_usage_tracker import (
+    licensed_items_checkouts as rut_licensed_items_checkouts,
+)
+from models_library.api_schemas_webserver import (
+    licensed_items_checkouts as webserver_licensed_items_checkouts,
+)
+from models_library.licensed_items import LicensedItemID
+from models_library.products import ProductName
+from models_library.resource_tracker import ServiceRunId
+from models_library.resource_tracker_licensed_items_checkouts import (
+    LicensedItemCheckoutID,
+)
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker import (
+    licensed_items_checkouts,
+)
+
+from ..rabbitmq import get_rabbitmq_rpc_client
+from ..users.api import get_user
+from ..wallets.api import get_wallet_by_user
+
+
+async def checkout_licensed_item_for_wallet(
+    app: web.Application,
+    licensed_item_id: LicensedItemID,
```

Comment:
TIP: enforce key arguments `*,` and group arguments to "document" the context. For instance, here I would do as
```
async def checkout(
   app,
   *,
   # access context
   product_name: ProductName,
   user_id: UserID,
   wallet_id: WalletID,
   # checkout args
   service_run_id: ServiceRunId,
   num_of_seats: PositiveInt,
):
```